### PR TITLE
ToC Cleanup

### PR DESCRIPTION
**Pull request by maiyahershey**
Preliminary cleanup of the toc, suggested titles, merged files

### CODE REVIEW

**Review by sarmento — APPROVED** [review]
LGTM!
I didn't know we had a duplicate folder :)
I don't have visibility to whether we may be deleting something that we shouldn't but I am assuming you checked that we actually keep the latest of the 2 folders (the one over which we may have made little improvements).

**Comment by maiyahershey** [comment]
@sarmento I did judge based on the text diffs and what was included in the final ToC, but maybe they were duplicated to avoid conflicts. I realised I was making changes on the duplicate and this is why i wasn't able to see anything. @IvanPombo can you just make sure everything is there? i added a new folder now that keeps all the duplicate files in one place in case we need to refer back to them later on.

File: docs/introduction/simulators.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Simulators
+# Configurable Simulations
```
Comment (IvanPombo) [inline docs/introduction/simulators.md:1]: I think this section is more about the actual simulators, so wdyt of "Configure Simulators"?

**Review by IvanPombo — APPROVED** [review]
Nicely done @maiyahershey! Just confirmed and highlighted the files that can be deleted - all in the overview folder that are now on the duplicates_ - this was due to the merging process when we deployed everything to production in a hurry :|

Just left some small changes! LGTM!

**Review by sergiomsantos — APPROVED** [review]

File: docs/introduction/index.md [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+</div>
 
-More specifically, we will introduce you to:
 
-- [Tasks](./tasks). This is the core computational object of the platform. Your simulation requests are Tasks to be executed remotely on the computational infrastructure we make available
-- [Shared and Dedicated Resources](./computational_resources_overview). Tasks get executed on computational resources, which can be Shared by all users, or which can be Dedicated to singe users
-- [Storage and Data Flow](./data_flow). The platforms move a lot of data around. Your input files need to be sent to remote computational resources and the output of the simulations needs to be made available to you. 
-- [Infrastructure](./infrastructure). Under the hood, there is an infrastructure that provides computational power for executing your tasks. We currently use Google Cloud Platform, and very soon we will have other options available.
+1. Start and manage remote Virtual Machines (VMs), operating either independently 
+or collectively as an MPICluster, that are equipped with the pre-installed simulation
```
[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/introduction/index.md:14]: deal let me change the wording for this bit

File: docs/api_reference/faq.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# FAQ
+# Frequently Asked Questions (FAQs)
```
[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/api_reference/faq.md:1]: yes you're right, it looks too long in the preview. i'll change it to FAQs

File: docs/introduction/index.md [hunk 4]
```diff
@@ -1,23 +1,43 @@
-# Introduction
+# How it works
```
[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/introduction/index.md:1]: i think this bit is okay, but i see your point on the confusion. let me see what else could work

**Pull request merged**